src/weight/lily-weight.js

```diff
@@ -16,11 +16,11 @@ export function calculateLilyWeight(profile) {
   const skillLevels = skillOrder.map((key) => profile.levels[key].level);
   const skillXP = skillOrder.map((key) => profile.levels[key].xp);
 
-  const cataCompletions = getTierCompletions(profile.dungeons.catacombs.floors);
-  const masterCataCompletions = getTierCompletions(profile.dungeons.master_catacombs.floors);
-  const cataXP = profile.dungeons?.catacombs?.level.xp ?? 0;
+  const cataCompletions = getTierCompletions(profile.dungeons?.catacombs?.floors ?? {});
+  const masterCataCompletions = getTierCompletions(profile.dungeons?.master_catacombs?.floors ?? {});
+  const cataXP = profile.dungeons?.catacombs?.level?.xp ?? 0;
 
-  const slayerXP = slayerOrder.map((key) => profile.slayers?.[key]?.level.xp ?? 0);
+  const slayerXP = slayerOrder.map((key) => profile.slayers?.[key]?.level?.xp ?? 0);
```

Comment:
Is this really needed?

Reply:
Yes, you can test on profile DynanicEx

Reply:
https://sky.shiiyu.moe/stats/DynanicEx
![image](https://user-images.githubusercontent.com/44071655/142345368-9bc85a49-22ff-477f-8abf-1429247b1337.png)

Reply:
DynamicEx

Sorry, wrote it on the phone and autocorrect changed it